src/code/PreRender.c

```diff
@@ -545,7 +549,7 @@ void PreRender_AntiAliasFilter(PreRender* this, s32 x, s32 y) {
     s32 buffB[5 * 3];
     s32 xi;
     s32 yi;
-    s32 pad;
+    s32 temp;
```

Comment:
```suggestion
    s32 invCvg;
```

Reply:
inv is 1/ though I get what you mean

Reply:
It's an "additive inverse" which should be clear from the context